README.md

```diff
@@ -3,13 +3,35 @@
 - List 3 reasons why asymptotic analysis may be misleading with respect to
   actual performance in practice.
 
+  1. We drop the lower order terms when doing asymptotic analysis and these could have actual impact on prefomance in real practice. 
+  2. In asymptotic analiysis we treat all constants at the same rate. When in practice constants can take different amounts of time. Like comparing strings takes longer then comparing integers.
+  3. Input size also makes a difference. If you look that the lecture slides and see the graph for "Average case running times" you see that Timsort takes a longer amount of time to sort a list size of ≈62500 items then heap sort. Then the amount of time levels out between the two sorting types before timsort rises, timsort again take more time as the list size grows. This shows that when you take bounds for asymptotic analysis, it will not always reflect what the best algorithm is for each individuel case when using it in practice.
```

Comment:
Still not following you here. How the runtime behaves with respect to input size is exactly what asymptotic analysis characterizes, so I don't know what you mean here.